yt_dlp/extractor/bilibili.py

```diff
@@ -724,6 +718,7 @@ def _real_extract(self, url):
                 duration=traverse_obj(initial_state, ('videoData', 'duration', {int_or_none})),
                 __post_extractor=self.extract_comments(aid))
 
+        play_info = self._download_playinfo(video_id, cid, headers=headers, query={'try_look': 1})
```

Comment:
per https://github.com/yt-dlp/yt-dlp/pull/11734#issuecomment-2518613153, should we do this?
```suggestion
        play_info = None
        if self.is_logged_in:
            play_info = traverse_obj(
                self._search_json(r'window\.__playinfo__\s*=', webpage, 'play info', video_id, default=None),
                ('data', {dict}))
        if not play_info:
            play_info = self._download_playinfo(video_id, cid, headers=headers, query={'try_look': 1})
```